scripts/make_guide_images.ts

```diff
@@ -0,0 +1,48 @@
+import { GUIDES } from "../src/routes/guides/urls.ts"
```

Comment:
Is this something that can be fixed?
![image](https://github.com/user-attachments/assets/481a749a-076e-4cd0-878c-316a3fab130a)
Does it imply that the `.ts` should be removed?

JS/TS is not my field really, just highlighting what the tools point out.